include/platform-aws.h

```diff
@@ -58,4 +59,13 @@ struct ec2_platform_data *platform_aws_get_platform_entry(const char *platform_t
 							  struct ec2_platform_data *platform_data_list,
 							  size_t platform_data_len);
 
+class Aws : public Platform {
```

Comment:
I'd strongly encourage you to name this PlatformAWS, not AWS.  Much clearer on what this is.